tests/typecheck/models/test_meta_options.yml

```diff
@@ -66,7 +66,7 @@
             class Meta(TypedModelMeta):
                 abstract = 7  # E: Incompatible types in assignment (expression has type "int", base class "TypedModelMeta" defined the type as "bool")  [assignment]
                 verbose_name = ['test']  # E: Incompatible types in assignment (expression has type "List[str]", base class "TypedModelMeta" defined the type as "Union[str, _StrPromise]")  [assignment]
-                unique_together = {1: 2}  # E: Incompatible types in assignment (expression has type "Dict[int, int]", base class "TypedModelMeta" defined the type as "Union[Sequence[Sequence[str]], Sequence[str]]")  [assignment]
+                unique_together = {1: 2}  # E: Incompatible types in assignment (expression has type "Dict[int, int]", base class "TypedModelMeta" defined the type as "Sequence[Sequence[str]]")  [assignment]
```

Comment:
This one is a bit weird. I'm presuming that `Sequence[Sequence[str]] | Sequence[str]` simplifies to just `Sequence[Sequence[str]]`... Maybe something we can simplify in the few places we do it:

```
ext/django_stubs_ext/db/models/__init__.py:39:        unique_together: ClassVar[Sequence[Sequence[str]] | Sequence[str]]
ext/django_stubs_ext/db/models/__init__.py:40:        index_together: ClassVar[Sequence[Sequence[str]] | Sequence[str]]  # Deprecated in Django 4.2
```

Reply:
Yes, let's do it. Good catch!